subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/EmptySubscribableBuildActionRunnerRegistration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.internal.provider;
+
+import org.gradle.initialization.BuildEventConsumer;
+import org.gradle.internal.progress.BuildOperationListener;
+
+import java.util.Collections;
+
+public class EmptySubscribableBuildActionRunnerRegistration implements SubscribableBuildActionRunnerRegistration {
```

Comment:
Actually, `getAll` doesn't fail and can return an empty `List`. My mistake.
Will remove this type.